docs/faq.md

```diff
@@ -14,11 +14,11 @@ The *Volume Ratio* attribute of an AdnMuscle allows to simulate volume gain (vol
 
 ## How can I simulate skin?
 
-Skin can be simulated using the [AdnSkin](maya/skin) deformer. This deformer requires a reference mesh to drive the skin simulation (i.e. the fascia) and a mesh to apply the deformer to (skin mesh). A simple setup is explained [here](maya/simple_setup#adnskin).
+Skin can be simulated using the [AdnSkin](maya/skin) deformer. This deformer requires a target mesh or meshes to drive the skin simulation (i.e. the fascia) and a mesh to apply the deformer to (skin mesh). A simple setup is explained [here](maya/simple_setup#adnskin).
```

Comment:
Is it better to say "This deformer requires one or more target meshes..."?

I'm not sure, most probably @lucasferryinbibo will know more about it.

Reply:
Yes! what you're saying is more correct in my opinion. I would go for that change!

Reply:
Done. Also added this change to other parts where I wrote "target mesh or meshes".